### ISSUE TITLE
Duplicate latest failure on Indexer

### ISSUE THREAD

**Issue by aberrix** [theotherp/nzbhydra#366]
Version 0.2.147

I'm experiencing an issue with a duplicate indexer failure in my statuses page, I think it may be affecting my stats page and percentages.

Indexer is listed only once on the main stats page as well as the configuration and search pages. As far as it seems the indexer is only configured once.

Indexer statuses page:
Indexer listed twice with two different latest failure dates

omgwtfnzbs.org	2016-09-13 21:45
omgwtfnzbs.org	2016-08-12 14:06

Since the Indexer is only configured once (one instance) on my server so I don't know why it would be "holding onto" a previous failure like this? I know it's not exactly the end of the world, but it driving me crazy :) is there a way to fix/clear/reset this? I've tried all the normal restarting the service/etc with no luck. I can't delete and recreate the indexer config because its omgwtf (built-in)

Thanks in advance!

**Comment by theotherp** [comment]
I'll try and add some cleanup code.

**Comment by aberrix** [comment]
just a heads up, I'm still seeing this issue persist.

**Comment by theotherp** [comment]
Please send your debug infos.

**Comment by aberrix** [comment]
send, please confirm you received it.

**Comment by theotherp** [comment]
Nope, got nothing. It's theotherp@gmx.de

**Comment by theotherp** [comment]
Got it.

**Comment by theotherp** [comment]
That shouldn't even be possible... Alright.

**Comment by aberrix** [comment]
yeah, its not the end of the world, it only shows up there and not in my
other stats page - but I can't get rid of it and was just curious if you'd
seen it before.

ps. thank you so much for all you do, I love nzbhydra and have turned many
friends onto it, because ALL THE GRAPHS! ;)

**Comment by theotherp** [comment]
Should be fixed.
